<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShotgunPlaylistDialog</class>
 <widget class="QDialog" name="ShotgunPlaylistDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Shotgun Playlist</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Department:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="uiDepartmentDDL"/>
     </item>
     <item>
      <widget class="QToolButton" name="uiRefreshBTN">
       <property name="toolTip">
        <string>Refresh Playlists</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="uiFilterClosedCHK">
       <property name="toolTip">
        <string>Only looks up playlists that have not been tagged as 'closed' in Shotgun.</string>
       </property>
       <property name="text">
        <string>Open only</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="uiDateCHK">
       <property name="toolTip">
        <string>Filters out playlists that have been created before the given date.</string>
       </property>
       <property name="text">
        <string>Since:</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="uiPlaylistDATE">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Filter by:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="uiFilterTXT"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTreeWidget" name="uiPlaylistTREE">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>DPT</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>NAME</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DATE_TIME</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DESCRIPTION</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="uiCancelBTN">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="uiOkBTN">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>uiDateCHK</sender>
   <signal>toggled(bool)</signal>
   <receiver>uiPlaylistDATE</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
